src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor.cs

```diff
@@ -217,6 +217,26 @@ private void HideMessageBar(MouseEventArgs args)
         {
             IsMessageBarHidden = true;
         }
+        private string RatingChangedText = "";
+
+        private List<Person>[] people = new List<Person>[2];
+
+        protected override void OnInitialized()
+        {
+            base.OnInitialized();
+            //initialize people list
+            Person persons= new Person();
```

Comment:
the plural form of "Person" is "People" :)

Reply:
You're right 👍